FWCore/Services/web/serv.py

```diff
@@ -0,0 +1,39 @@
+from http.server import HTTPServer, BaseHTTPRequestHandler
```

Comment:
Please rename the file to `server.py` as that is clearer and also matches the documentation.